lib/pluginUtils.js

```diff
@@ -21,8 +21,9 @@ function getPluginName (func) {
   const keys = Object.keys(cache)
 
   for (let i = 0; i < keys.length; i++) {
-    if (cache[keys[i]].exports === func) {
-      return keys[i]
+    const key = keys[i]
```

Comment:
what is the optimization in here?

Reply:
Defining the key once and then using it multiple times should be faster than the program having to look for it again every time you want to use it.